specification/authorization/resource-manager/Microsoft.Authorization/preview/2020-10-01-preview/RoleAssignmentSchedule.json

```diff
@@ -184,7 +184,7 @@
             "Device"
           ],
           "x-ms-enum": {
-            "name": "PrincipalType",
+            "name": "principalTypePrivate",
```

Comment:
Temporary fix for collision in enums
will be discussed internally before undrafting this PR

Reply:
@amanswaika

Reply:
resolved offline, closing